ObjectMapper/Core/Operators.swift

```diff
@@ -345,4 +345,35 @@ public func <- <T: Mappable>(inout left: Array<T>!, right: Map) {
 	} else {
 		ToJSON.optionalObjectArray(left, key: right.currentKey!, dictionary: &right.JSONDictionary)
 	}
-}
\ No newline at end of file
+}
+
+
+// MARK:- Set of Mappable objects - Set<T: Mappable>
+
+/// Array of Mappable objects
+public func <- <T: Mappable>(inout left: Set<T>, right: Map) {
```

Comment:
These functions should constrain the generic parameter as `<T: Mappable where T: Hashable>`.